From fef640c0e34cc33827e0778543dc2a72df26def7 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Wed, 17 Sep 2014 13:58:58 -0400
Subject: [PATCH] vm_cgroup: fix warning about int fcn w/o return value
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

In file included from kernel/fork.c:74:0:
include/linux/vm_cgroup.h: In function ‘vm_cgroup_charge_shmem’:
include/linux/vm_cgroup.h:69:1: warning: no return statement in function returning non-void [-Wreturn-type]

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 include/linux/vm_cgroup.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/vm_cgroup.h b/include/linux/vm_cgroup.h
index 6f16b57567ae..3bbe697cea73 100644
--- a/include/linux/vm_cgroup.h
+++ b/include/linux/vm_cgroup.h
@@ -66,6 +66,7 @@ static inline void shmem_release_vm_cgroup(struct shmem_inode_info *info)
 static inline int vm_cgroup_charge_shmem(struct shmem_inode_info *info,
 					 unsigned long nr_pages)
 {
+	return 0;
 }
 
 static inline void vm_cgroup_uncharge_shmem(struct shmem_inode_info *info,
-- 
1.8.1.2

